lib/bunyan.js

```diff
@@ -1184,6 +1168,52 @@ function safeCycles() {
     };
 }
 
+// A JSON stringifier that handles cycles safely - tracks seen values in a Set.
+function safeCyclesSet() {
+    var seen = new Set();
+    return function (key, val) {
+        if (!val || typeof (val) !== 'object') {
+            return val;
+        }
+        if (seen.has(val)) {
+            return '[Circular]';
+        }
+        seen.add(val);
+        return val;
+    };
+}
+
+// A JSON stringifier that handles cycles safely.
```

Comment:
style nit: The rest of the module tends to `/* ... */` style comments if multi-line.